examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py

```diff
@@ -146,8 +145,20 @@ def __process_worker_register(self, worker_id, input_value_json, response):
                 "Worker Id already exists in the database. \
                  Hence invalid parameter")
         return response
+
 # ------------------------------------------------------------------------------------------------
+    def validate_input_worker_status(self, input_json):
+        try:
+            worker_status = input_json["params"]["status"]
+            status = int(worker_status)
+            if status < 0 or status > 4:
```

Comment:
Can we use enum for status instead of "4"

Reply:
Done